modules/eco-node-health-check-operator-creating-node-health-check.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-health-check-operator.adoc
```

Comment:
File path relative to the module location:
```suggestion
// * ../nodes/nodes/eco-node-health-check-operator.adoc
```

Reply:
> File path relative to the module location:

Thanks for your feedback Jesse. I have applied the update as you suggest.